tensilelite/Tensile/SolutionStructs/Solution.py

```diff
@@ -2657,8 +2673,14 @@ def subCheckLdsBlockSizePerPad(tc, idx):
       state["LdsOffsetMetadata"] = state["LdsOffsetA"] + state["LdsNumElementsAlignedA"]
       state["LdsOffsetB"] = state["LdsOffsetMetadata"] + state["LdsNumElementsAlignedMetadata"]
 
-      offsetBlk = state["LdsOffsetB"] +  ldsNumBytesAlignedB
-      if offsetBlk > 0:
+      offsetBlk = state["LdsOffsetB"] + ldsNumBytesAlignedB
```

Comment:
is it possible to use offsetBlk to replace int(2**(math.ceil(math.log(offsetBlk, 2)))) for all case??
swapValue = 1_LDS + offset_A
swapValue = swapValue xor offset_A 
offset_A_blk = offset_A xor swapValue 
offset_A  = offset_A_blk xor swapValue 
is it correct?

Reply:
> is it possible to use offsetBlk to replace int(2**(math.ceil(math.log(offsetBlk, 2)))) for all case??

Sorry, I don't understand the question? Do you mean always use store the swapValue in a register?

> swapValue = 1_LDS + offset_A
swapValue = swapValue xor offset_A
offset_A_blk = offset_A xor swapValue
offset_A = offset_A_blk xor swapValue

Yes, that is correct. Just checking, in your comment, `offset_A` is first buffer and `offset_A_blk` is second buffer right?